### DISCUSSION TITLE
using function-based views

### DISCUSSION THREAD

**Discussion by ccurvey** [codingjoe/joeflow#75] [Ideas]
I'm a Djangosaur, and I haven't learned class-based views yet.    Is there a way to use function-based views with joeflow?  

(And please let me know if there's a better location to ask questions.)

**Comment by codingjoe** [comment]
Hi @ccurvey,

loved the intro 🦖!
Not really no. At least not at the moment. I'd need to invest some effort to make it work.
A decorator could be used to inject the necessary functionality.

I am pretty behind on my notifications as is. Sooo, If you want to pitch in to contribute this feature, I wouldn't say no.

Cheers!
Joe
